BankApp/Account.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BankApp
+{
+    class Account
+    {
+        public string Name { get; private set; }
+        public Guid AccountNumber { get; }
```

Comment:
No need to name the field AccountName, when its a field of an Account class. Name would be enough. Same for Balance.